life_hc/MySQL/General/fn_updatePartial.sqf

```diff
@@ -8,29 +8,36 @@
     Takes partial data of a player and updates it, this is meant to be
     less network intensive towards data flowing through it for updates.
 */
-private ["_uid","_side","_value","_value1","_value2","_mode","_query"];
-_uid = [_this,0,"",[""]] call BIS_fnc_param;
-_side = [_this,1,sideUnknown,[civilian]] call BIS_fnc_param;
-_mode = [_this,3,-1,[0]] call BIS_fnc_param;
+params [
+    ["_uid","",[""]],
+    ["_side",sideUnknown,[civilian]],
+    ["_mode",-1,[0]]
+];
 
 if (_uid isEqualTo "" || _side isEqualTo sideUnknown) exitWith {}; //Bad.
-_query = "";
+private _query = "";
 
 switch (_mode) do {
     case 0: {
-        _value = [_this,2,0,[0]] call BIS_fnc_param;
+        params [
```

Comment:
yikes, this ain't how params work, you're wanting the 4th index, yet your selecting the 1st again here (getting uid) etc